.github/workflows/image.yml

```diff
@@ -112,3 +111,40 @@ jobs:
           provenance: mode=max
           sbom: true
           push: true
+
+  create-manifest:
+    name: Create multi-platform manifest
+    needs: build-main-or-release
+    if: |
+      github.repository_owner == 'getsentry'
+      && github.event_name == 'push'
+      && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) || startsWith(github.ref, 'refs/heads/release/'))
+
+    permissions:
+      contents: read
+      packages: write
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
+
+      - name: Login to GHCR
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3
+        with:
+          username: sentrybuilder
+          password: ${{ secrets.DOCKER_HUB_RW_TOKEN }}
+
+      - name: Create and push multi-platform manifest
+        run: |
+          ./scripts/create-manifest.sh \
+            "${{ github.repository }}" \
+            "${{ github.sha }}" \
+            "${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}"
```

Comment:
What does this end up evaluating to? `1/0` `true/false`?

Reply:
I think `true`? 
https://docs.github.com/en/actions/reference/evaluate-expressions-in-workflows-and-actions#operators